src/dune_rules/melange/melange_rules.ml

```diff
@@ -597,6 +597,7 @@ let setup_js_rules_libraries =
             in
             cmj_includes ~requires_link ~scope lib_config
           in
+          let output = output_of_lib ~target_dir vlib in
```

Comment:
fix explanation: the `output` in scope was for `output_of_lib ~target_dir lib` instead of `vlib`.